frontend/src/App.js

```diff
@@ -114,6 +126,7 @@ class App extends Component {
           <Route exact path="/home" component={Home} />
           <Route exact path="/campaigns" component={Campaigns} />
           <Route exact path="/users" component={Users} />
+          <Route exact path="/edit/:uid" component={UserEdit} />
```

Comment:
I tried to make this route `/users/:uid/edit` or `/users/edit/:uid`, but because of how the `/users/:uid` container works neither of those were possible.

Reply:
Fine for now, but once this is merged let's open a new PR for how we want to merge this functionality into the /dashboard page.